diff --git a/packager/src/JSTransformer/worker/extract-dependencies.js b/packager/src/JSTransformer/worker/extract-dependencies.js
index 1ce908b..1ddd75e 100644
--- a/packager/src/JSTransformer/worker/extract-dependencies.js
+++ b/packager/src/JSTransformer/worker/extract-dependencies.js
@@ -33,12 +33,8 @@ function extractDependencies(code: string) {
       const node = path.node;
       const callee = node.callee;
       const arg = node.arguments[0];
-      if (
-        callee.type !== 'Identifier' ||
-        callee.name !== 'require' ||
-        !arg ||
-        arg.type !== 'StringLiteral'
-      ) {
+      if (!((callee.type === 'Identifier' && callee.name === 'require' && arg && arg.type === 'StringLiteral') ||                                                                                                                      
+         (callee.type === 'MemberExpression' && callee.object && callee.object.name === "goog" && callee.property && callee.property.name == "require" && arg && arg.type === 'StringLiteral'))){
         return;
       }
       dependencyOffsets.push(arg.start);
